client/my-sites/marketing/ultimate-traffic-guide/index.js

```diff
@@ -0,0 +1,316 @@
+					{
+						args: { currentYear: new Date().getFullYear() },
+					}
+				) }
+			</p>
+			<p>
+				{ translate(
+					'Here are just a few of the topics we cover in {{i}}The Ultimate Traffic Guide:{{/i}}',
+					{
+						components: {
+							i: <i />,
+						},
+					}
+				) }
+			</p>
+			<p>
+				<strong>{ translate( 'The complete SEO checklist for your website (page 53)' ) }</strong>
+				<br></br>
+				{ translate(
+					'Following this simple 7-step checklist will help you get your site featured at the top of search engine results for your most important keywords (and help you avoid the most common pitfalls that you absolutely must avoid if you want to rank well).'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate(
+						'Why email is STILL a must-have marketing channel (hint: it’s simple, free, and effective) (page 88)'
+					) }
+				</strong>
+				<br></br>
+				{ translate(
+					'This guide will teach you everything you need to know: from which email marketing tools you should use, to how to collect email addresses and write great subject lines, and more.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate( 'Harness the power of social media to expand your reach (page 65)' ) }
+				</strong>
+				<br></br>
+				{ translate(
+					'Discover the best ways to leverage social media channels like FaceBook, Twitter, and LinkedIn to connect with your audience and turn them into raving fans.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate(
+						'How to build a simple “traffic magnet” to instantly attract the right audience (page 7)'
+					) }
+				</strong>
+				<br></br>
+				{ translate(
+					'It’s so effective you don’t even have to come close to a “perfect” site before you launch… even the ugliest site can drive an endless stream of business if you get this right!'
+				) }
+			</p>
+			{ isLoading ? (
+				<Placeholder />
+			) : (
+				<p>
+					{ translate(
+						'If you’ve spent any time looking for marketing help, you know that marketing guides like this often cost %(referenceCost)s or more. But you won’t have to pay anywhere near that amount.',
+						{
+							args: { referenceCost: displayReferenceCost },
+						}
+					) }
+				</p>
+			) }
+
+			{ isLoading ? (
+				<Placeholder />
+			) : (
+				<p>
+					<strong>
+						{ translate(
+							'In fact, if you act today you’ll pay just %(cost)s for The Ultimate Traffic Guide.',
+							{
+								args: { cost: displayCost },
+							}
+						) }
+					</strong>
+				</p>
+			) }
+
+			{ cta() }
+		</>
+	);
+
+	const treatmentVariation = () => (
+		<>
+			<h1 className="ultimate-traffic-guide__header">
+				{ translate(
+					'Discover the most important traffic secrets used by our most successful customers'
+				) }
+			</h1>
+			<h2 className="ultimate-traffic-guide__sub-header">
+				{ translate( 'Claim your copy of the Ultimate Traffic Guide today' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Claim your copy of the <br /><em>Ultimate Traffic Guide</em>' )` *ES Score: 7*